<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductMenuDialog</class>
 <widget class="QDialog" name="ProductMenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="customTitleBar">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" name="customTitleLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="titleSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="customTitle">
        <property name="text">
         <string>Barcode Product Management</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="titleSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QPushButton" name="customCloseBtn">
        <property name="toolTip">
         <string>Close</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabAdd">
      <attribute name="title">
       <string>ADD</string>
      </attribute>
    <layout class="QVBoxLayout" name="addLayout">
       <item>
        <widget class="QLabel" name="addSubHeaderLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Add New Product</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="add_formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="addProductCodeLabel">
           <property name="text">
            <string>Product Code*</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="addProductCodeLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="addProductNameLabel">
           <property name="text">
            <string>Product Name*</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="addProductNameLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="addCategoryLabel">
           <property name="text">
            <string>Category*</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="addCategoryComboBox">
           <item>
            <property name="text">
             <string>Other</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Electronics</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Food</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Apparel</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="addCostPriceLabel">
           <property name="text">
            <string>Cost Price</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="addCostPriceLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="addSellingPriceLabel">
           <property name="text">
            <string>Selling Price*</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="addUnitLabel">
           <property name="text">
            <string>Unit</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="addUnitLineEdit">
           <property name="text">
            <string>EACH</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="addSupplierLabel">
           <property name="text">
            <string>Supplier</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="addSupplierLineEdit"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="addSellingPriceLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="addStatusLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="add_actionBtns">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddOk">
           <property name="text">
            <string>ADD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddCancel">
           <property name="text">
            <string>CANCEL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRemove">
      <attribute name="title">
       <string>REMOVE</string>
      </attribute>
    <layout class="QVBoxLayout" name="removeLayout">
       <item>
        <widget class="QLabel" name="removeSubHeaderLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Remove Product</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="remove_formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="removeProductCodeLabel">
           <property name="text">
            <string>Product Code</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="removeProductNameLabel">
           <property name="text">
            <string>Product Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="removeCategoryLabel">
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="removeCostPriceLabel">
           <property name="text">
            <string>Cost Price</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="removeSellingPriceLabel">
           <property name="text">
            <string>Selling Price</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="removeUnitLabel">
           <property name="text">
            <string>Unit</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="removeSupplierLabel">
           <property name="text">
            <string>Supplier</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="removeLastUpdatedLabel">
           <property name="text">
            <string>Last updated</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="removeProductCodeLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="removeProductNameComboBox"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="removeCategoryLineEdit"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="removeCostPriceLineEdit"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="removeSellingPriceLineEdit"/>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="removeSupplieLineEdit"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="removeUnitLineEdit"/>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="removeLastUpdatedLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="removeStatusLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="remove_actionBtns">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveOk">
           <property name="text">
            <string>DELETE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveCancel">
           <property name="text">
            <string>CANCEL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUpdate">
      <attribute name="title">
       <string>UPDATE / VIEW</string>
      </attribute>
    <layout class="QVBoxLayout" name="updateLayout">
       <item>
        <widget class="QLabel" name="updateSubHeaderLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Update / View Product</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="update_formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="updateProductCodeLabel">
           <property name="text">
            <string>Product Code *</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="updateProductCodeLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="updateProductNameLabel">
           <property name="text">
            <string>Product Name *</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="updateProductNameComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="updateCategoryLabel">
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="updateCategoryComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="updateCostPriceLabel">
           <property name="text">
            <string>Cost Price</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="updateCostPriceLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="updateSellingPriceLabel">
           <property name="text">
            <string>Selling Price *</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="updateSellingPriceLineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="updateUnitLabel">
           <property name="text">
            <string>Unit</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="updateUnitLineEdit">
           <property name="text">
            <string>EACH</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="updateSupplierLabel">
           <property name="text">
            <string>Supplier</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="updateSupplierLineEdit"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="updateLastUpdatedLabel">
           <property name="text">
            <string>Last updated</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="updateLastUpdatedLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="updateStatusLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="update_actionBtns">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdateOk">
           <property name="text">
            <string>UPDATE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdateCancel">
           <property name="text">
            <string>CANCEL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
